ZaeXT/test-scripts/test_backend.py

```
@@ -0,0 +1,285 @@
+        for event in client.events():
+            if event.event == 'message':
+                print(event.data, end="", flush=True)
+                full_response += event.data
+        print("\n") # 换行
+        
+        if "print(\"HelloWorld!\")" in full_response:
+             print_success("流式聊天测试成功，并收到预期内容")
+        else:
+             print_fail("流式聊天未收到预期内容")
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)